apps/codeforafrica/src/components/ProjectPageHeader/ProjectPageHeader.js

```diff
@@ -38,7 +39,7 @@ const ProjectPageHeaderRoot = styled(Card, {
   padding: `${theme.spacing(2.5)} 0`,
   [theme.breakpoints.up("sm")]: {
     alignItems: "flex-start",
-    flexDirection: "row",
+    flexDirection: "row-reverse",
```

Comment:
Is this a design change?

Reply:
This is needed to fix [this](https://github.com/CodeForAfrica/ui/pull/610#issuecomment-1740351323)